docs/deploying a sample wildfly application.md

```diff
@@ -0,0 +1,144 @@
+        mariadb      172.30.47.201:5000/openshift/mariadb      latest,10.1                  7 days ago
+        mongodb      172.30.47.201:5000/openshift/mongodb      latest,3.2,2.6 + 1 more...   7 days ago
+        mysql        172.30.47.201:5000/openshift/mysql        latest,5.6,5.5               7 days ago
+        nodejs       172.30.47.201:5000/openshift/nodejs       4,0.10,latest                7 days ago
+        perl         172.30.47.201:5000/openshift/perl         5.20,5.16,latest             7 days ago
+        php          172.30.47.201:5000/openshift/php          latest,5.6,5.5               7 days ago
+        postgresql   172.30.47.201:5000/openshift/postgresql   latest,9.5,9.4 + 1 more...   7 days ago
+        python       172.30.47.201:5000/openshift/python       latest,3.5,3.4 + 2 more...   7 days ago
+        ruby         172.30.47.201:5000/openshift/ruby         latest,2.3,2.2 + 1 more...   7 days ago
+        wildfly      172.30.47.201:5000/openshift/wildfly      10.0,9.0,8.1 + 1 more...     7 days ago
+
+  You will notice that the WildFly imagestream already exists in your OpenShift Origin cluster. If a WildFly imagestream is unavailable you can [create an Imagestream](https://docs.openshift.org/latest/dev_guide/managing_images.html) to build your application.
+
+  You can also use available templates in OpenShift Origin to deploy an application, refer to OpenShift Origin docs for further details on [templates](https://docs.openshift.org/latest/dev_guide/app_tutorials/quickstarts.html). In this case we use the available imagestream in OpenShift Origin.
+
+1. Create your new application on OpenShift as follows:
+
+        $ oc new-app --name=myapp wildfly:latest~https://github.com/<GitHub username>/os-sample-java-web
+        --> Found image ece19b5 (7 days old) in image stream "wildfly" in project "openshift" under tag "latest" for "wildfly:latest"
+
+            WildFly 10.0.0.Final
+            --------------------
+            Platform for building and running JEE applications on WildFly 10.0.0.Final
+
+            Tags: builder, wildfly, wildfly10
+
+            * A source build using source code from https://github.com/<GitHub username>/os-sample-java-web will be created
+              * The resulting image will be pushed to image stream "myapp:latest"
+              * Use 'start-build' to trigger a new build
+            * This image will be deployed in deployment config "myapp"
+            * Port 8080/tcp will be load balanced by service "myapp"
+              * Other containers can access this service through the hostname "myapp"
+
+        --> Creating resources with label app=myapp ...
+            imagestream "myapp" created
+            buildconfig "myapp" created
+            deploymentconfig "myapp" created
+            service "myapp" created
+        --> Success
+            Build scheduled, use 'oc logs -f bc/myapp' to track its progress.
+            Run 'oc status' to view your app.
+
```

Comment:
@hferentschik  so what I understand is after this step we need to add a step here like:
"Monitor your build to verify that your build has been deployed successfully by using: 

```
$ oc describe build
Name:		myapp-1
Namespace:	myproject
Created:	5 hours ago
Labels:		app=myapp
		buildconfig=myapp
		openshift.io/build-config.name=myapp
		openshift.io/build.start-policy=Serial
Annotations:	openshift.io/build-config.name=myapp
		openshift.io/build.number=1
		openshift.io/build.pod-name=myapp-1-build

Status:		Complete
Started:	Fri, 03 Feb 2017 13:11:10 IST
Duration:	7m34s
Build Config:	myapp
Build Pod:	myapp-1-build

Strategy:		Source
URL:			https://github.com/<GitHub username>/os-sample-java-web
Commit:			74cdd67 (README added)
Author/Committer:	<username> / GitHub
From Image:		DockerImage openshift/wildfly-101-centos7@sha256:269d97c9a9e6f38b7908b1a3053aed1b6f76a053760582567b5521424f162290
Output to:		ImageStreamTag myapp:latest
Push Secret:		builder-dockercfg-yi2g3

Build trigger cause:	Build configuration change

No events.
```
Alternatively you can use `$ oc logs build/myapp-1` too to monitor your build."

Since the output for `$ oc logs build/myapp-1` command is humungous, maybe we can just leave it that, wdyt? Please let me know if this is what you had in  mind.